src/python/ddapp/ikplanner.py

```diff
@@ -986,9 +988,10 @@ def planGraspOrbitReachPlan(self, startPose, side, graspFrame, constraints=None,
     def addPose(self, pose, poseName):
         self.jointController.addPose(poseName, pose)
 
-        if self.pushToMatlab:
+        if (self.pushToMatlab is True or self.pushToMatlabIK is True):
             self.ikServer.sendPoseToServer(pose, poseName)
-        else:
+        #else:
+	if (self.pushToMatlab is False or self.pushToMatlabIK is False):
```

Comment:
seems that either flag being false implies publishing to an external planner.  perhaps the flag could be renamed to make it more clear that it is used to decide when you publish to an external plan.